### ISSUE TITLE
configure cocoaPods

### ISSUE THREAD

**Issue by udithaPS** [jharwig/PPSSignatureView#21]
I,m new to iOS development. I didn't use cocoapods before. can you please give me complete work floor about configuring this project with cocoaPods.

Initially I got this issue..

PhaseScriptExecution Check\ Pods\ Manifest.lock /Users/entt/Library/Developer/Xcode/DerivedData/PPSSignatureDemo-aegeuwydbgumskbugijmhqyhohfc/Build/Intermediates/PPSSignatureDemo.build/Debug-iphonesimulator/PPSSignatureDemo.build/Script-51C5B5620D3B4693B8ED7D34.sh
    cd /Users/entt/Desktop/editors/PPSSignatureView-master/Example
    /bin/sh -c /Users/entt/Library/Developer/Xcode/DerivedData/PPSSignatureDemo-aegeuwydbgumskbugijmhqyhohfc/Build/Intermediates/PPSSignatureDemo.build/Debug-iphonesimulator/PPSSignatureDemo.build/Script-51C5B5620D3B4693B8ED7D34.sh

diff: /../Podfile.lock: No such file or directory
diff: /Manifest.lock: No such file or directory
error: The sandbox is not in sync with the Podfile.lock. Run 'pod install' or update your CocoaPods installation.

**Comment by MosheBerman** [comment]
First you need to install cocoapods. To do that, you need to run `sudo gem install cocoapods` in Terminal. 

Next, you need to make sure you are in the correct directory. That's usually the one that has a podfile and your Xcode project in it. That's `cd PATH_TO_THE_EXAMPLE_DIRECTORY`, wherever you might have downloaded the repository to.

Finally, run `pod install`. This will find and install the cocoapods in your pod file in the current directory. It will also generate an `xcworkspace` file for you to use instead of your project.

So, to summarize:

```
$> sudo gem install cocoapods
$> cd PATH_TO_EXAMPLE_DIRECTORY
$> pod install
```

Then you should be good to go.

![screen shot 2015-12-10 at 11 19 00 am](https://cloud.githubusercontent.com/assets/427452/11721145/63c2caf8-9f30-11e5-8d02-7388cb18eb4b.png)

**Comment by CavanLife** [comment]
$> pod install
Analyzing dependencies
[!] The dependency `PPSSignatureView (from `../PPSSignatureView.podspec`)` is not used in any concrete target.

how can i do this?

**Comment by CavanLife** [comment]
Yeah, i have resolved this issue.  Modify the podfile content with `target 'PPSSignatureDemo' do

pod 'PPSSignatureView', :path => '../PPSSignatureView.podspec'

end`

then, pod install.

**Comment by CavanLife** [comment]
Yeah, i have resolved this issue. Modify the podfile content with 

> target 'PPSSignatureDemo' do

>pod 'PPSSignatureView', :path => '../PPSSignatureView.podspec'

>end

then, pod install.
